OWNERS

```diff
@@ -1,18 +1,15 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - anishasthana
-  - hemajv
   - HumairAK
   - tumido
-  - martinpovolny
-  - 4n4nd
-  - durandom
 
 reviewers:
+  - durandom
   - anishasthana
   - hemajv
-  - HumairAK
-  - tumido
   - martinpovolny
   - 4n4nd
+
+labels:
+  - sig/documentation
```

Comment:
This PR is against the `operate-first/apps` root folder. I don't think `sig/documentation` is the right label. :) I think you can drop this bit here.

Reply:
Looking at [this](https://github.com/kubernetes/kubernetes/blob/master/OWNERS#L33) it looks like you can apply labels to a file type regex. Maybe we can have one for *.md and `sig/documentation` 

I may not be fully understanding `filters` however.

Reply:
Other cool ones would be if the `cluster-scope` is changed, a "cluster-scope` label or something similar would be nice, or if it's the odh path have it labelled `odh`

I could be getting carried away here.

Reply:
I think we can also do this with separate `OWNERS` files per each sub folder or via [filters](https://www.kubernetes.dev/docs/guide/owners/#filters)

Reply:
all the things said are right and good, feel free to open a PR against this PR ;)